csharp/test/security/CWE-1004/CookieHttpOnlyFalseAspNetCore/CookieBuilder/HttpOnly.qlref

```diff
@@ -0,0 +1 @@
+experimental/Security Features/CWE-1004/CookieWithoutHttpOnly.ql
```

Comment:
@michaelnebel do these paths need to be updated?

Reply:
Btw. what do you think about the query IDs?

Reply:
Im ok using `githubsecuritylab` but maybe we want to use something related to community packs, @GeekMasher any thoughts?

Reply:
Great. I have just prefixed the query IDs with `githubsecuritylab`. Then lets go with that.
Should we just go ahead and get the queries merged?